x/airdrop/types/msgs.go

```diff
@@ -6,9 +6,13 @@ import (
 	"github.com/ojo-network/ojo/util/checkers"
 )
 
+const (
+	TypeMsgSetParams    = "set_params"
+	TypeMsgClaimAirdrop = "claim_airdrop"
+)
+
```

Comment:
Do we need these?
```suggestion
```

Reply:
Looks like I'm not use the `TypeMsgSetParams` variable anymore but the `TypeMsgClaimAirdrop` one is used here in the sim tests: https://github.com/ojo-network/ojo/pull/175/files#diff-02c479d5110d899cf359516a66dd9ec8f1e2961bd7731eefe1f5561ed2bcec4aR65 I saw another sim test example use a constant for the message name and thought it was a good idea.

Reply:
removed the first one: https://github.com/ojo-network/ojo/pull/175/commits/88b1b593801fd6eccbb89069b11405c12a869f4b